### ISSUE TITLE
Security Scheme related params are not shown in sidebar panel for not-custom schemas

### ISSUE THREAD

**Issue by psotres** [mulesoft/api-console#228]
Any header or query param defined as part of a security scheme on its describedBy  section is only included in the sidebar panel if the schema is a custom one. This should also be done for traditional schemas. See example RAML:

```
#%RAML 0.8 BasicSecurityScheme
type: Basic Authentication
description: |
  Basic auth together with extra parameters description
describedBy:
  headers:
    Authorization:
      description: |
        The Authorization field value consists of credentials containing the authentication information of the user agent for the realm of the resource being requested.
  queryParameters:
    extraToken:
      description: |
        Extra parameter needed for authorization
```

In the previous example the API is using basic-auth as the auth schema, but Authorization and extraToken are not shown. The correct behaviour should be to hide Authorization from the panel (as it's handled by the API console itself but it's interesting for documentation purposes) but still show the extraToken query parameter.

**Comment by sichvoge** [comment]
related to https://github.com/mulesoft/api-console/issues/308. @psotres does your PR still work with the newest version? can you merge the master into your branch and find out?

**Comment by nmarinel01** [comment]
Hi @psotres ,
this seems to be fixed using latest Api-Console release (3.0.16). 
![image](https://cloud.githubusercontent.com/assets/19980188/24669741/f228e796-1941-11e7-8f92-f7611429198c.png)

Can you verify and reopen if applies ? 
Regards !
